### ISSUE TITLE
[SODA] Enable review workflow cannot be selected from the keyboard

### ISSUE THREAD

**Issue by aaron-collier** [sul-dlss/hungry-hungry-hippo#901]
When creating a collection, on the Workflow tab, the Enable Review Workflow options cannot be selected from the keyboard. There is no way to perform the function using only the keyboard on the same screen or on a qualifying conforming alternate version.

Soda recommendation:
```
RULE : 
Functionality MUST be available using the keyboard, unless the functionality cannot be accomplished in any known way using a keyboard.

HOW TO FIX: 
Fix this issue by ensuring the component can be used by the keyboard. This includes ensuring controls are:
1. Focusable with a keyboard (i.e. can be reached using the Tab key or Arrow keys on a keyboard), and
2. Operable with a keyboard (e.g. links are activated with the Enter key, buttons are activated with Enter and Spacebar, checkboxes are activated with Spacebar, dragging and swiping actions have a keyboard alternative, etc.)

The best way to do this is to use native HTML elements when HTML equivalents exist (e.g. anchor, button, select, text area, and input elements). When custom (non native HTML) controls have to be used, you must ensure they can be focused with a keyboard (often using tabindex="0") and operated with a keyboard (using JavaScript event handlers). The ARIA Authoring Practices Guide provides information on expected keyboard interaction patterns for custom controls and widgets and working examples (see References below).

REFERENCE: 
Deque University: https://dequeuniversity.com/class/input-methods/keyboard-input/functionality
ARIA Authoring Practices Guide Patterns: https://www.w3.org/WAI/ARIA/apg/patterns/

BACKGROUND: 
Some people cannot use a mouse due to vision or motor disabilities. Content that can be operated with a mouse must also be made operable with a keyboard. When content is operable through a keyboard, it becomes operable by a variety of assistive technologies such as speech input software, sip-and-puff software, on-screen keyboards, scanning software, and alternate keyboards.
```

See issue id 2269 in https://docs.google.com/spreadsheets/d/1x0uBBDe3T8O0HNCs74OdVKQc8tQQNvOV_f-C6IS44IQ/edit?gid=1050712151#gid=1050712151
